docs/refarch/boundedContexts/scheduling/index.md

```diff
@@ -1,23 +1,133 @@
+
+#### Flow Diagram
+
+![Delete (Recurring) Reminder](./assets/schedulingDeleteRecurringReminder_20211021.png)
+>
 
 <!-- Footnotes themselves at the bottom. -->
 ## Notes
 
-[^1]: Common Interfaces: [Mojaloop Common Interface List](../../commonInterfaces.md)
+#### Create Reminder - Required data
+
+The Create Reminder request needs to include the following data:
+
+| Data | Description |
+| --- | ---- |
+| *Identifier* | name/id |
+| *Cron Definition* | recurring?, time interval? |
+| *Trigger Transport* | HTTP Callback/Event; Callback URL or Event Topic |
+| *Special Payload* | opaque for the scheduling BC |
+| *Recovery Conditions* | retry, reschedule, abort, abandon |
+| *Alerts* | notification, logging on exceptions |
+| *Actions* | register of automatable/schedulable BC processes |
+
+#### Scheduling BC - Requirements
+
+The Scheduling BC must meet the following requirements:
+
+* Reminders must be triggered once only
+
+* BC must keep track of triggered Reminders
+
+* BC must keep track of Create/Read/Delete actions
+
+    * Updates will be facilitated through Delete/Create actions as noted in the [Use Case Available UCs](#use-cases) list
+
+* Job batches
+
+* Offer multiple interface options (gRPC, REST, HTTP, etc.)
+
+* Reminders should be triggered with an HTTP callback, not a gRPC call, or to a specific topic
+
+* It should have no ability to process logic external to the Scheduling BC itself
+
+* Make use of Linux-based UTC timestamps only in order to avoid synchronization issues
+
+***Note:*** *It is assumed that the underlying system will keep perfect time.*
+
+#### Scheduling BC - Outstanding requirements
+
+Access requirements for the Scheduling BC still require definition.
+
+#### Scheduling BC - Exceptions
+
+* Malformed instructions
+    * Invalid date/time, including times in the past
+    * Invalid BC or command
+* Failed execution (identified through call-back)
+* Insufficient authority for the Client BC to perform the CRD operation
+* Failure to process/execute Reminder
+
+#### Questions
+
+A number of questions came up during the Reference Architecture Work Sessions and, as some were felt to have potential value from which others would benefit, we have included them below:
+
+* After the scheduled task has been initiated, does the Scheduling BC remain responsible for tracking its progress?
+
+    * Answer: No. When the Reminder is due, it is communicated to the Client BC using the precribed method, and thereafter, the Scheduling BC responsibility for the reminder is passed back to the Client BC.
+
+* Is the Client BC or the person who scheduled a Reminder noted as the "User" by the Scheduling BC?  IOW, whose ID is stamped on the process audit trail?
+
+    * Answer: This should be determined by the Client BC, based on its action on receipt of the Reminder.
+
+[^1]: For further information please view: [Link Name](https://...)
```

Comment:
Should this not be removed since we do not have any `[^1]` references?

Reply:
removed